{
  "Header": {
    "Original": "CVE-2023-46445 (An issue in AsyncSSH before 2.14.1 allows attackers to control the ext ...)",
    "ID": "CVE-2023-46445",
    "Description": "(An issue in AsyncSSH before 2.14.1 allows attackers to control the ext ...)"
  },
  "Annotations": [
    {
      "Original": "{DLA-3899-1}",
      "Type": "xref",
      "Bugs": [
        "DLA-3899-1"
      ]
    },
    {
      "Original": "- python-asyncssh 2.15.0-1 (bug #1056000)",
      "Type": "package",
      "Package": "python-asyncssh",
      "Kind": "fixed",
      "Version": "2.15.0-1",
      "BugNo": 1056000
    },
    {
      "Original": "[bookworm] - python-asyncssh 2.10.1-2+deb12u2",
      "Type": "package",
      "Release": "bookworm",
      "Package": "python-asyncssh",
      "Kind": "fixed",
      "Version": "2.10.1-2+deb12u2"
    },
    {
      "Original": "[buster] - python-asyncssh \u003cno-dsa\u003e (Minor issue)",
      "Type": "package",
      "Release": "buster",
      "Package": "python-asyncssh",
      "Kind": "no-dsa",
      "Description": "Minor issue"
    },
    {
      "Original": "NOTE: https://github.com/ronf/asyncssh/security/advisories/GHSA-cfc2-wr2v-gxm5",
      "Type": "NOTE",
      "Description": "https://github.com/ronf/asyncssh/security/advisories/GHSA-cfc2-wr2v-gxm5"
    },
    {
      "Original": "NOTE: https://github.com/ronf/asyncssh/commit/83e43f5ea3470a8617fc388c72b062c7136efd7e (v2.14.1)",
      "Type": "NOTE",
      "Description": "https://github.com/ronf/asyncssh/commit/83e43f5ea3470a8617fc388c72b062c7136efd7e (v2.14.1)"
    },
    {
      "Original": "NOTE: https://terrapin-attack.com/",
      "Type": "NOTE",
      "Description": "https://terrapin-attack.com/"
    }
  ]
}